frontend/src/v5/ui/controls/formSelect/formSelect.component.tsx

```diff
@@ -44,6 +46,7 @@ export const FormSelect = ({
 		<Controller
 			control={control}
 			name={name}
+			defaultValue={defaultValue}
```

Comment:
what happens if defaultValue is undefined?

Reply:
the input switches from 'uncontrolled' to 'control' as soon as it changes its value printing an error in the console